shigeifinder/1.3.5/Dockerfile

```diff
@@ -0,0 +1,178 @@
+# global arg variables
+ARG SHIGEIFINDER_VER="1.3.5"
+ARG SAMTOOLS_VER="1.10"
+ARG BWA_VER="0.7.17"
+
+### start of builder stage ###
+FROM ubuntu:focal as builder
+
+# re-instantiate these variables
+ARG SHIGEIFINDER_VER
+ARG SAMTOOLS_VER
+ARG BWA_VER
+
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="ShigEiFinder"
+LABEL software.version="${SHIGEIFINDER_VER}"
+LABEL description="Cluster informed Shigella and EIEC serotyping tool from Illumina reads and assemblies"
+LABEL website="https://github.com/LanLab/ShigEiFinder"
+LABEL license="https://github.com/LanLab/ShigEiFinder/blob/main/LICENSE"
+LABEL maintainer="Curtis Kapsak"
+LABEL maintainer.email="curtis.kapsak@theiagen.com"
```

Comment:
These labels are not making it to the test stage of the build. I don't have a lot of time to experiment with this, but I think you want to put these in `app` instead.

Reply:
got it thank you. I will make the adjustment shortly